### ISSUE TITLE
WCAG Name, Role Value - Role is wrong for panel edit menu items

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#106223]
**Steps to reproduce**:

TAB to "Add visualization" button and hit ENTER key. "Select data source" popup will appear. Select "Azure monitor" default data source. 
5. TAB to "Save dashboard" button and hit ENTER key. Changes will be saved, now TAB to "Back to dashboard" button and hit ENTER key. 
Tab navigates till “Menu button presents in the panel. 

1. Create a new Dashboard
2. Add Visualization (new panel)
3. Turn on Screen Narrator (Windows Narrator, NVDA, etc.)
4. Tab to thee dot menu in top right corner
5. Notice the role description by the narrator as you navigate through the panel Menu and sub menus.  Some are links when they should be menu items.  Each entry in that list should have a role defined and the role should be accurate (e.g. menu instead of link, etc).  Make sure to look through sub-menu items to ensure they are correct as well.

![Image](https://github.com/user-attachments/assets/3bca14e0-5738-48c3-a7f5-07153b11d17f)

**Actual Result**:

Observe that Narrator Screen Reader is announcing incorrect role information for menu items present on dashboard.

**Expected Result**

Each entry in the panel edit menu should have correct role defined.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Environment**:
- Grafana version: 12.0.2
- Data source type & version: n/a
- User OS & Browser:
    Chromium Edge: Version 137.0.3269.1 (Official build) dev (64-bit) 
    Edition Windows 11 Pro 
    Version 24H2 
    OS build 26100.3476
    Screen reader used: - Narrator/NVDA  
    NVDA Version: 2025.3(2025.3.0.33433) 
- Others:

**Comment by JonCole** [comment]
FYI - this is a Sev2 work item from our perspective

**Comment by joshhunt** [comment]
For context, we adjusted this in https://github.com/grafana/grafana/pull/86886
